test/nodes/test_reader.py

```diff
@@ -38,11 +38,13 @@ def no_answer_reader(request):
         )
 
 
+@pytest.mark.integration
```

Comment:
`integration` because `reader` model needs to be downloaded during initialization?

Reply:
Yes, the `reader` fixture would load the model.